<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Epidiagnostics Group</title> <meta name="author" content="Thomas Pisanic"/> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "/> <meta name="keywords" content="epigenetics, DNA methylation, Pisanic, diagnostics"/> <meta property="og:site_name" content="Epidiagnostics Group"/> <meta property="og:type" content="website"/> <meta property="og:title" content="Epidiagnostics Group | about"/> <meta property="og:url" content="https://hunky-d0ry.github.io/"/> <meta property="og:description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "/> <meta property="og:image" content="/assets/img/epidiagnostics-logo.png"/> <meta property="og:locale" content="en"/> <meta name="twitter:card" content="summary"/> <meta name="twitter:title" content="about"/> <meta name="twitter:description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "/> <meta name="twitter:image" content="/assets/img/epidiagnostics-logo.png"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/favicon.ico"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://hunky-d0ry.github.io/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container px-md-3"> <a href="/" class=""> <img src="/assets/img/logos/epidiagnostics-logo.png" height="45" alt="Epidiagnostics Group"> </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/team.html">team</a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/funding.html">funding</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/collaborators.html">collaborators</a> </div> </li> <div class="toggle-container"> <a id="light-toggle"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </a> </div> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <a href="/members/pisanic"> <img class="img-fluid z-depth-1 rounded" src="/assets/img/team/prof_pic.png" alt="Thomas R. Pisanic II, PhD"> </a> <div class="address"> <p> <a href="/members/pisanic"> <span class="name">Thomas R. Pisanic II, PhD</span> </a> <span class="data"> <br>Epidiagnostics Group Investigator<br>Associate Research Professor <br> <a href="https://breakthroughcancer.org/" target="_blank" rel="noopener noreferrer">Break <i>Through</i> Cancer</a> Scientist </span> </p> <p class="post">Institute for NanoBioTechnology<br> Shaffer 200D<br> 3400 N Charles St<br> Baltimore, MD 21218 </p> <div> <a href="https://inbt.jhu.edu" target="_blank" rel="noopener noreferrer"><img class="layout_commented" style="margin-bottom: 20px" img src="/assets/img/logos/INBT-logo.png" height="60"></a> <a href="https://breakthroughcancer.org" target="_blank" rel="noopener noreferrer"><img class="layout_commented" style="margin-bottom: 20px" img src="/assets/img/logos/BTC-logo.png" height="60"></a> </div> </div> </div> <div class="clearfix"> <div class="about-logo"> <img style="padding-bottom: 1rem; margin-right: 2rem" src="/assets/img/logos/epidiagnostics-logo.png" height="150"> </div> <p>Welcome to to the website of Dr. Thomas Pisanic and the Epidiagnostics Group at Johns Hopkins University! I work with the <a href="https://me.jhu.edu/thwang/" target="_blank" rel="noopener noreferrer">Wang laboratory</a> and an interdisciplinary <a href="/team/">team of JHU researchers</a> to gain deeper insight into the epigenetics of carcinogenesis and apply this knowledge toward the development of novel microfluidic molecular analysis platforms for cancer diagnostic applications. Our “epidiagnostics” team is comprised of faculty with a wide range of complementary expertise, including: <a href="/members/Pisanic">Dr. Thomas Pisanic </a> (epigenetics, biomarker discovery and assay development), <a href="https://engineering.jhu.edu/faculty/tza-huei-jeff-wang/" target="_blank" rel="noopener noreferrer"> Dr. Tza-Huei Wang </a>(bioengineering and microfluidic technologies), <a href="https://www.hopkinsmedicine.org/profiles/details/hariharan-easwaran" target="_blank" rel="noopener noreferrer">Dr. Hari Easwaran </a> (genetics and epigenetics of cancer and aging) and <a href="https://www.hopkinsmedicine.org/profiles/details/leslie-cope" target="_blank" rel="noopener noreferrer">Dr. Leslie Cope</a> (biostatistics and bioinformatics). Together, we work with numerous <a href="/collaborators"> collaborators</a> from Johns Hopkins and beyond to apply our epigenetic methods to resolve manifold challenges in various areas of cancer research.</p> </div> <div class="news"> <h2>news</h2> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" width="150 px">May 16, 2023</th> <td> <a href="https://inbt.jhu.edu/working-to-detect-ovarian-cancer-earlier/" target="_blank" rel="noopener noreferrer">Working to Detect Ovarian Cancer Earlier</a> (INBT Press Release) </td> </tr> <tr> <th scope="row" width="150 px">Apr 25, 2023</th> <td> <a href="https://breakthroughcancer.org/2023/04/break-through-cancer-a-new-research-consortium-names-first-class-of-scientists-designated-as-catalysts-for-innovation-and-collaboration/" target="_blank" rel="noopener noreferrer">Break Through Cancer, A New Research Consortium, Names First Class of Scientists Designated as Catalysts for Innovation and Collaboration</a> (BTC Press Release) </td> </tr> <tr> <th scope="row" width="150 px">Jul 1, 2022</th> <td> Dr. Pisanic appointed as Associate Research Professor at INBT with joint appointment in the SOM Department of Oncology. </td> </tr> <tr> <th scope="row" width="150 px">Apr 22, 2022</th> <td> <a href="https://breakthroughcancer.org/2022/04/break-through-cancer-announces-50m-in-grants-to-empower-researchers-from-five-top-cancer-research-centers-to-work-as-one/" target="_blank" rel="noopener noreferrer">Break Through Cancer Announces $50M in Grants to Support Team Science!</a> </td> </tr> <tr> <th scope="row" width="150 px">Mar 1, 2022</th> <td> Dr. Pisanic appointed as Break Through Cancer Scientist! </td> </tr> </table> </div> </div> <div class="publications"> <h2>selected publications</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cancers</abbr></div> <div id="RN74" class="col-sm-9"> <div class="title">Current and Emerging Methods for Ovarian Cancer Screening and Diagnostics: A Comprehensive Review</div> <div class="author">Liberto, Juliane M., Chen, Sheng-Yin, Shih, Ie-Ming, Wang, Tza-Huei, Wang, Tian-Li, and Pisanic, Thomas R. </div> <div class="periodical"> <em>Cancers</em>, 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.mdpi.com/2072-6694/14/12/2885" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Ovarian high-grade serous carcinoma (HGSC) has a 5-year survival rate of less than 50%, making it one of the most lethal gynecological cancers for women in the developed world today. Delayed presentation of clinical symptoms and late-stage diagnosis drive the high mortality rate of this disease. Early detection is associated with significant improvements in survival, however, screening in the general population is currently not recommended at this time due to a notable lack of sensitive and specific biomarkers for early-stage disease. In this review, we provide an overview of the current landscape of ovarian cancer diagnostics, emphasizing emerging methodologies for the non-invasive detection of HGSC.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN74</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liberto, Juliane M. and Chen, Sheng-Yin and Shih, Ie-Ming and Wang, Tza-Huei and Wang, Tian-Li and Pisanic, Thomas R.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Current and Emerging Methods for Ovarian Cancer Screening and Diagnostics: A Comprehensive Review}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cancers}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2885}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2072-6694}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.mdpi.com/2072-6694/14/12/2885}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Epigenetics</abbr></div> <div id="RN44" class="col-sm-9"> <div class="title">Leveraging locus-specific epigenetic heterogeneity to improve the performance of blood-based DNA methylation biomarkers</div> <div class="author">Miller, Brendan F.,  Pisanic, Thomas R., Margolin, Gennady, Petrykowska, Hanna M., Athamanolap, Pornpat, Osei-Tutu, Akosua, Annunziata, Christina M., Wang, Tza-Huei, and Elnitski, Laura </div> <div class="periodical"> <em>Clinical Epigenetics</em>, 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-020-00939-w" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Aberrant DNA methylation is commonly heralded as a promising cancer biomarker; however, its inherently stochastic nature often leads to variable methylation patterns that can complicate the use of methylation biomarkers for clinical diagnostics, particularly in dilute samples such as liquid biopsies. Here, we present a methylation density binary classifier, a statistical method for leveraging differential heterogeneous methylation to predict and optimize the performance of methylation biomarkers for clinical applications. We first developed and tested the classifier using methylation density profiles derived from reduced representation bisulfite sequencing reads of ovarian carcinoma at ZNF154, a recurrently methylated locus in multiple cancer types. We then used in silico simulations to predict the performance of the classifier in liquid biopsies and validated these predictions using quasi-digital melt curve analysis (DREAMing) of circulating cell-free DNA from individuals with versus without ovarian carcinoma. We found good agreement between predicted and observed classifier performance, and further demonstrated that implementation of this approach with ZNF154 outperformed CA-125 for use in etiologically-diverse ovarian cancer types. Our results indicate that methylation density profiles can be exploited to predict and facilitate implementation of methylation biomarkers for clinical applications, and that ZNF154 methylation shows promise as a clinically-useful biomarker for ovarian cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN44</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Miller, Brendan F. and Pisanic, Thomas R. and Margolin, Gennady and Petrykowska, Hanna M. and Athamanolap, Pornpat and Osei-Tutu, Akosua and Annunziata, Christina M. and Wang, Tza-Huei and Elnitski, Laura}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Leveraging locus-specific epigenetic heterogeneity to improve the performance of blood-based DNA methylation biomarkers}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Epigenetics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13148-020-00939-w}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/10.1101/579839v2}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Cancer Res.</abbr></div> <div id="RN50" class="col-sm-9"> <div class="title">Methylomic Landscapes of Ovarian Cancer Precursor Lesions</div> <div class="author"> Pisanic, Thomas R., Wang, Yeh, Sun, Hanru, Considine, Michael, Li, Lihong, Wang, Tza-Huei, Wang, Tian-Li, and Shih, Ie-Ming </div> <div class="periodical"> <em>Clinical Cancer Research</em>, 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://aacrjournals.org/clincancerres/article/26/23/6310/83008/Methylomic-Landscapes-of-Ovarian-Cancer-Precursor" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Purpose: The current paradigm in the development of high-grade serous ovarian carcinoma (HGSC) proposes that the majority of HGSCs arise from precursor serous tubal intraepithelial carcinoma (STIC) lesions of the fallopian tube. Here we survey genome-wide methylation in HGSC precursor lesions to identify genomic regions that exhibit high-specificity differential hypermethylation for potential use as biomarkers for detecting STIC and HGSC at stages when curative intervention likely remains feasible.Experimental Design: We first identified quality control criteria for performing reliable methylomic analysis of DNA-limited tubal precursor lesions with the Illumina Infinium MethylationEPIC array. We then used this platform to compare genome-wide methylation among 12 STICs with paired adjacent-normal epithelia, one p53 signature lesion and two samples of concurrent HGSC. The resulting methylomic data were analyzed by unsupervised hierarchical clustering and multidimensional analysis. Regions of high-confidence STIC-specific differential hypermethylation were identified using selective bioinformatic criteria and compared with published MethylationEPIC data from 23 HGSC tumors and 11 healthy fallopian tube mucosae.Results: Unsupervised analysis showed that STICs largely clustered with HGSCs, but were clearly distinct from adjacent-normal fallopian tube epithelia. Forty-two genomic regions exhibited high-confidence STIC-specific differential hypermethylation, of which 17 (40.5%) directly overlapped with HGSC-specific differentially methylated regions. Methylation at these shared loci was able to completely distinguish STIC and HGSC samples from normal and adjacent-normal specimens.Conclusions: Our results suggest that most STICs are epigenetically similar to HGSCs and share regions of differential hypermethylation that warrant further evaluation for potential use as biomarkers for early detection of ovarian HGSC.See related commentary by Ishak and De Carvalho, p. 6083</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN50</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, Thomas R. and Wang, Yeh and Sun, Hanru and Considine, Michael and Li, Lihong and Wang, Tza-Huei and Wang, Tian-Li and Shih, Ie-Ming}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Methylomic Landscapes of Ovarian Cancer Precursor Lesions}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Cancer Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{26}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{23}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{6310-6320}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1158/1078-0432.Ccr-20-0270}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Sci. Adv.</abbr></div> <div id="RN39" class="col-sm-9"> <div class="title">Facile profiling of molecular heterogeneity by microfluidic digital melt</div> <div class="author">O’Keefe, Christine M.,  Pisanic, Thomas R., Zec, Helena, Overman, Michael J., Herman, James G., and Wang, Tza-Huei </div> <div class="periodical"> <em>Science Advances</em>, 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/10.1126/sciadv.aat6459" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>This work presents a digital microfluidic platform called HYPER-Melt (high-density profiling and enumeration by melt) for highly parallelized copy-by-copy DNA molecular profiling. HYPER-Melt provides a facile means of detecting and assessing sequence variations of thousands of individual DNA molecules through digitization in a nanowell microchip array, allowing amplification and interrogation of individual template molecules by detecting HRM fluorescence changes due to sequence-dependent denaturation. As a model application, HYPER-Melt is used here for the detection and assessment of intermolecular heterogeneity of DNA methylation within the promoters of classical tumor suppressor genes. The capabilities of this platform are validated through serial dilutions of mixed epialleles, with demonstrated detection limits as low as 1 methylated variant in 2 million unmethylated templates (0.00005%) of a classic tumor suppressor gene, CDKN2A (p14ARF). The clinical potential of the platform is demonstrated using a digital assay for NDRG4, a tumor suppressor gene that is commonly methylated in colorectal cancer, in liquid biopsies of healthy and colorectal cancer patients. Overall, the platform provides the depth of information, simplicity of use, and single-molecule sensitivity necessary for rapid assessment of intermolecular variation contributing to genetic and epigenetic heterogeneity for challenging applications in embryogenesis, carcinogenesis, and rare biomarker detection.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN39</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{O’Keefe, Christine M. and Pisanic, Thomas R. and Zec, Helena and Overman, Michael J. and Herman, James G. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Facile profiling of molecular heterogeneity by microfluidic digital melt}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science Advances}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{eaat6459}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/sciadv.aat6459}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nucleic Acids Res.</abbr></div> <div id="RN75" class="col-sm-9"> <div class="title">DREAMing: a simple and ultrasensitive method for assessing intratumor epigenetic heterogeneity directly from liquid biopsies</div> <div class="author"> Pisanic II, Thomas R., Athamanolap, Pornpat, Poh, Weijie, Chen, Chen, Hulbert, Alicia, Brock, Malcolm V., Herman, James G., and Wang, Tza-Huei </div> <div class="periodical"> <em>Nucleic Acids Research</em>, 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1093/nar/gkv795" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Many cancers comprise heterogeneous populations of cells at primary and metastatic sites throughout the body. The presence or emergence of distinct subclones with drug-resistant genetic and epigenetic phenotypes within these populations can greatly complicate therapeutic intervention. Liquid biopsies of peripheral blood from cancer patients have been suggested as an ideal means of sampling intratumor genetic and epigenetic heterogeneity for diagnostics, monitoring and therapeutic guidance. However, current molecular diagnostic and sequencing methods are not well suited to the routine assessment of epigenetic heterogeneity in difficult samples such as liquid biopsies that contain intrinsically low fractional concentrations of circulating tumor DNA (ctDNA) and rare epigenetic subclonal populations. Here we report an alternative approach, deemed DREAMing (Discrimination of Rare EpiAlleles by Melt), which uses semi-limiting dilution and precise melt curve analysis to distinguish and enumerate individual copies of epiallelic species at single-CpG-site resolution in fractions as low as 0.005%, providing facile and inexpensive ultrasensitive assessment of locus-specific epigenetic heterogeneity directly from liquid biopsies. The technique is demonstrated here for the evaluation of epigenetic heterogeneity at p14ARF and BRCA1 gene-promoter loci in liquid biopsies obtained from patients in association with non-small cell lung cancer (NSCLC) and myelodysplastic/myeloproliferative neoplasms (MDS/MPN), respectively.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN75</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic II, Thomas R. and Athamanolap, Pornpat and Poh, Weijie and Chen, Chen and Hulbert, Alicia and Brock, Malcolm V. and Herman, James G. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{DREAMing: a simple and ultrasensitive method for assessing intratumor epigenetic heterogeneity directly from liquid biopsies}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nucleic Acids Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{43}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e154-e154}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0305-1048}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/nar/gkv795}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1093/nar/gkv795}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%74%70%69%73%61%6E%69%63@%6A%68%75.%65%64%75" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0001-5796-0836" title="ORCID" target="_blank" rel="noopener noreferrer"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=W7_NTgMAAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a> <a href="https://www.researchgate.net/profile/Thomas-Pisanic/" title="ResearchGate" target="_blank" rel="noopener noreferrer"><i class="ai ai-researchgate"></i></a> <a href="https://www.linkedin.com/in/tpisanic" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a> <a href="https://hunky-d0ry.github.io/feed.xml" title="RSS Feed"><i class="fas fa-rss-square"></i></a> </div> <div class="contact-note"> </div> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Thomas Pisanic. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Last updated: June 26, 2023. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>