<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Epidiagnostics Group</title> <meta name="author" content="Thomas Pisanic"/> <meta name="description" content="Publications from the Epidiagnostics Group."/> <meta name="keywords" content="epigenetics, DNA methylation, Pisanic, diagnostics"/> <meta property="og:site_name" content="Epidiagnostics Group"/> <meta property="og:type" content="website"/> <meta property="og:title" content="Epidiagnostics Group | publications"/> <meta property="og:url" content="https://hunky-d0ry.github.io/publications/"/> <meta property="og:description" content="Publications from the Epidiagnostics Group."/> <meta property="og:image" content="/assets/img/epidiagnostics-logo.png"/> <meta property="og:locale" content="en"/> <meta name="twitter:card" content="summary"/> <meta name="twitter:title" content="publications"/> <meta name="twitter:description" content="Publications from the Epidiagnostics Group."/> <meta name="twitter:image" content="/assets/img/epidiagnostics-logo.png"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/favicon.ico"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://hunky-d0ry.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container px-md-3"> <a href="/" class=""> <img src="/assets/img/logos/epidiagnostics-logo.png" height="45" alt="Epidiagnostics Group"> </a> <a class="navbar-brand title font-weight-lighter" href="https://hunky-d0ry.github.io/"></a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/team.html">team</a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/funding.html">funding</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/collaborators.html">collaborators</a> </div> </li> <div class="toggle-container"> <a id="light-toggle"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </a> </div> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description" style="font-size: 1.2rem">Publications from the Epidiagnostics Group.</p> </header> <article> <div class="publications"> <h2 class="year">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cancer Res.</abbr></div> <div id="RN77" class="col-sm-9"> <div class="title">Abstract 6510: REM-DREAMing: Low-cost digital microfluidic analysis of DNA methylation heterogeneity for enhanced, liquid biopsy-based detection of early-stage lung cancer</div> <div class="author">Zhao, Yang, O’Keefe, Christine M., Herman, James G.,  Pisanic, Thomas R., and Wang, Tza-Huei </div> <div class="periodical"> <em>Cancer Research</em>, 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1158/1538-7445.AM2023-6510" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>We report the results of a study exploring the ability to exploit molecular heterogeneity in DNA methylation for improving the performance of liquid biopsy-based screening for early-stage (I &amp; II) non-small cell lung cancer (NSCLC). Annual low-dose CT (LDCT) screening is currently recommended for adults aged 50 or older who are at high-risk of developing lung cancer. While this approach has resulted in improvements in survival, the false positive rate of lung nodules detection by LDCT remains over 95%, leading to unnecessary invasive follow-up procedures and further points to the need for new, complementary methods to improve diagnostics performance and reduce patient risk. DNA methylation biomarkers have demonstrated considerable potential as tumor-specific biomarkers for blood-based detection of early-stage NSCLC. Nonetheless, cell-free DNA (cfDNA) assessment techniques, such as methylation-specific PCR (MSP) or bisulfite sequencing, have limited sensitivity to assess epigenetic heterogeneity of rare epiallelic variants in a cost-effective manner. Here we reported a new platform, named REM-DREAMing (Ratiometric-Encoded Multiplex Discrimination of Rare EpiAlleles by Melt), which provides a simple, low-cost solution for multiplexed assessment of loci-specific DNA methylation heterogeneity at single molecule sensitivity. The microfluidic nanoarray contains four independent but identical 10,040 nanowell modules. Methylation biomarkers are differentiated by a ratiometric fluorescence scheme and the assessment of individual epiallele species of each locus are achieved through digitization in the nanoarray and precise high-resolution melt (HRM) analysis. In this study, we explore the potential utility of REM-DREAMing as a complementary assay for improving LDCT screening of NSCLC by testing a cohort of 48 clinical samples (28 cancer and 20 control samples) of low-volume liquid biopsy specimens from patients with CT-scan indeterminant pulmonary nodules. A machine learning algorithm incorporating logistic regression models with leave-one-out cross validation was developed to identify a proper methylation density threshold of each biomarker in the panel. Evaluation of the receiver operation characteristic (ROC) curve yielded an area under the curve (AUC) of 0.97 (95% CI, 0.94-1) with 93% sensitivity at 95% specificity for the REM-DREAMing assay, compared with 93% sensitivity at 62% specificity achieved using a traditional, MSP-based approach. These results suggest that the assessment of intermolecular epigenetic heterogeneity can provide superior clinical performance for cfDNA methylation and noninvasive detection of early-stage NSCLC, in particular. Our results warrant further investigation in a larger sample cohort to validate its utility for improving routine screening of NSCLC in high-risk populations.Citation Format: Yang Zhao, Christine M. O’Keefe, James G. Herman, Thomas R. Pisanic, Tza-Huei Wang. REM-DREAMing: Low-cost digital microfluidic analysis of DNA methylation heterogeneity for enhanced, liquid biopsy-based detection of early-stage lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2023; Part 1 (Regular and Invited Abstracts); 2023 Apr 14-19; Orlando, FL. Philadelphia (PA): AACR; Cancer Res 2023;83(7_Suppl):Abstract nr 6510.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN77</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhao, Yang and O’Keefe, Christine M. and Herman, James G. and Pisanic, Thomas R. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Abstract 6510: REM-DREAMing: Low-cost digital microfluidic analysis of DNA methylation heterogeneity for enhanced, liquid biopsy-based detection of early-stage lung cancer}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cancer Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{83}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7_Supplement}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{6510-6510}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0008-5472}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1158/1538-7445.Am2023-6510}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1158/1538-7445.AM2023-6510}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Advanced Science</abbr></div> <div id="RN80" class="col-sm-9"> <div class="title">Multiplex Digital Methylation-Specific PCR for Noninvasive Screening of Lung Cancer</div> <div class="author">Zhao, Yang, O’Keefe, Christine M., Hsieh, Kuangwen, Cope, Leslie, Joyce, Sonali C.,  Pisanic, Thomas R., Herman, James G., and Wang, Tza-Huei </div> <div class="periodical"> <em>Advanced Science</em>, 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/advs.202206518" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Abstract There remains tremendous interest in developing liquid biopsy assays for detection of cancer-specific alterations, such as mutations and DNA methylation, in cell-free DNA (cfDNA) obtained through noninvasive blood draws. However, liquid biopsy analysis is often challenging due to exceedingly low fractions of circulating tumor DNA (ctDNA), necessitating the use of extended tumor biomarker panels. While multiplexed PCR strategies provide advantages such as higher throughput, their implementation is often hindered by challenges such as primer-dimers and PCR competition. Alternatively, digital PCR (dPCR) approaches generally offer superior performance, but with constrained multiplexing capability. This paper describes development and validation of the first multiplex digital methylation-specific PCR (mdMSP) platform for simultaneous analysis of four methylation biomarkers for liquid-biopsy-based detection of non-small cell lung cancer (NSCLC). mdMSP employs a microfluidic device containing four independent, but identical modules, housing a total of 40 160 nanowells. Analytical validation of the mdMSP platform demonstrates multiplex detection at analytical specificities as low as 0.0005%. The clinical utility of mdMSP is also demonstrated in a cohort of 72 clinical samples of low-volume liquid biopsy specimens from patients with computed tomography (CT)-scan indeterminant pulmonary nodules, exhibiting superior clinical performance when compared to traditional MSP assays for noninvasive detection of early-stage NSCLC.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN80</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhao, Yang and O'Keefe, Christine M. and Hsieh, Kuangwen and Cope, Leslie and Joyce, Sonali C. and Pisanic, Thomas R. and Herman, James G. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multiplex Digital Methylation-Specific PCR for Noninvasive Screening of Lung Cancer}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Advanced Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2206518}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2198-3844}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1002/advs.202206518}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/advs.202206518}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">MicroTAS 2022</abbr></div> <div id="RN79" class="col-sm-9"> <div class="title">A microfluidic platform for quantitative multiplex profiling of DNA methylation biomarkers. </div> <div class="author">Zhao, Y, O’Keefe, C. M., Herman, J.,  Pisanic, T.R., and Wang, T. H. </div> <div class="periodical"> <em>The 26th International Conference on Miniaturized Systems for Chemistry and Life Sciences (MicroTAS 2022)</em>, 2022 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN79</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhao, Y and O’Keefe, C. M. and Herman, J. and Pisanic, T.R. and Wang, T. H.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A microfluidic platform for quantitative multiplex profiling of DNA methylation biomarkers. }</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The 26th International Conference on Miniaturized Systems for Chemistry and Life Sciences (MicroTAS 2022)}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cancers</abbr></div> <div id="RN74" class="col-sm-9"> <div class="title">Current and Emerging Methods for Ovarian Cancer Screening and Diagnostics: A Comprehensive Review</div> <div class="author">Liberto, Juliane M., Chen, Sheng-Yin, Shih, Ie-Ming, Wang, Tza-Huei, Wang, Tian-Li, and Pisanic, Thomas R. </div> <div class="periodical"> <em>Cancers</em>, 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.mdpi.com/2072-6694/14/12/2885" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Ovarian high-grade serous carcinoma (HGSC) has a 5-year survival rate of less than 50%, making it one of the most lethal gynecological cancers for women in the developed world today. Delayed presentation of clinical symptoms and late-stage diagnosis drive the high mortality rate of this disease. Early detection is associated with significant improvements in survival, however, screening in the general population is currently not recommended at this time due to a notable lack of sensitive and specific biomarkers for early-stage disease. In this review, we provide an overview of the current landscape of ovarian cancer diagnostics, emphasizing emerging methodologies for the non-invasive detection of HGSC.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN74</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liberto, Juliane M. and Chen, Sheng-Yin and Shih, Ie-Ming and Wang, Tza-Huei and Wang, Tian-Li and Pisanic, Thomas R.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Current and Emerging Methods for Ovarian Cancer Screening and Diagnostics: A Comprehensive Review}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cancers}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2885}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2072-6694}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.mdpi.com/2072-6694/14/12/2885}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Biosens. Bioelect.</abbr></div> <div id="RN68" class="col-sm-9"> <div class="title">Magnetofluidic immuno-PCR for point-of-care COVID-19 serological testing</div> <div class="author">Zhang, Pengfei, Chen, Liben, Hu, Jiumei, Trick, Alexander Y., Chen, Fan-En, Hsieh, Kuangwen, Zhao, Yang, Coleman, Branch, Kruczynski, Kate,  Pisanic, Thomas R., Heaney, Christopher D., Clarke, William A., and Wang, Tza-Huei </div> <div class="periodical"> <em>Biosensors and Bioelectronics</em>, 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S095656632100693X" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Serological tests play an important role in the fight against Coronavirus Disease 2019 (COVID-19), including monitoring the dynamic immune response after vaccination, identifying past infection and determining community infection rate. Conventional methods for serological testing, such as enzyme-linked immunosorbent assays and chemiluminescence immunoassays, provide reliable and sensitive antibody detection but require sophisticated laboratory infrastructure and/or lengthy assay time. Conversely, lateral flow immunoassays are suitable for rapid point-of-care tests but have limited sensitivity. Here, we describe the development of a rapid and sensitive magnetofluidic immuno-PCR platform that can address the current gap in point-of-care serological testing for COVID-19. Our magnetofluidic immuno-PCR platform automates a magnetic bead-based, single-binding, and one-wash immuno-PCR assay in a palm-sized magnetofluidic device and delivers results in ∼30 min. In the device, a programmable magnetic arm attracts and transports magnetically-captured antibodies through assay reagents pre-loaded in a companion plastic cartridge, and a miniaturized thermocycler and a fluorescence detector perform immuno-PCR to detect the antibodies. We evaluated our magnetofluidic immuno-PCR with 108 clinical serum/plasma samples and achieved 93.8% (45/48) sensitivity and 98.3% (59/60) specificity, demonstrating its potential as a rapid and sensitive point-of-care serological test for COVID-19.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN68</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Pengfei and Chen, Liben and Hu, Jiumei and Trick, Alexander Y. and Chen, Fan-En and Hsieh, Kuangwen and Zhao, Yang and Coleman, Branch and Kruczynski, Kate and Pisanic, Thomas R. and Heaney, Christopher D. and Clarke, William A. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Magnetofluidic immuno-PCR for point-of-care COVID-19 serological testing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biosensors and Bioelectronics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{195}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{113656}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0956-5663}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.bios.2021.113656}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S095656632100693X}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">SLAS Techn.</abbr></div> <div id="RN72" class="col-sm-9"> <div class="title">High-throughput sample processing for methylation analysis in an automated, enclosed environment</div> <div class="author">Stark, Alejandro,  Pisanic, Thomas R., Herman, James G., and Wang, Tza-Huei </div> <div class="periodical"> <em>SLAS Technology</em>, 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2472630321000248" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Variation in methylcytosine is perhaps the most well-studied epigenetic mechanism of gene regulation. Methods that have been developed and implemented for assessing DNA methylation require sample DNA to be extracted, purified and chemically-processed through bisulfite conversion before downstream analysis. While some automated solutions exist for each of these individual process steps, a fully integrated solution for accomplishing the entire process in a high-throughput manner has yet to be demonstrated. Thus, sample processing methods still require numerous manual steps that may reduce sample throughput and precision, while increasing the risk of contamination and human error. In this work, we present an integrated, automated solution for performing the entire sample preparation process, including DNA extraction, purification, bisulfite conversion and PCR plate preparation within in an enclosed environment. The method employs silica-coated magnetic particles that eliminate the need for a centrifuge or vacuum manifold, thereby reducing the complexity and cost of the required automation platform. Toward this end, we also compare commercial DNA extraction and bisulfite conversion kits to identify a protocol suitable for automation to significantly improve genomic and bisulfite-treated DNA yields over manufacturer protocols. Overall, this research demonstrated development of an automated protocol that offers the ability to generate high-quality, bisulfite-treated DNA samples in a high-throughput and clean environment with minimal user intervention and comparable yields to manual processing.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN72</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Stark, Alejandro and Pisanic, Thomas R. and Herman, James G. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High-throughput sample processing for methylation analysis in an automated, enclosed environment}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{SLAS Technology}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2472-6303}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.slast.2021.12.002}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S2472630321000248}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Anal. Chem.</abbr></div> <div id="RN54" class="col-sm-9"> <div class="title">Ligation-Enabled Fluorescence-Coding PCR for High-Dimensional Fluorescence-Based Nucleic Acid Detection</div> <div class="author">Park, Joon Soo,  Pisanic, Thomas, Zhang, Ye, and Wang, Tza-Huei </div> <div class="periodical"> <em>Analytical Chemistry</em>, 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1021/acs.analchem.0c04221" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Polymerase chain reaction (PCR) is by far the most commonly used method of nucleic acid amplification and has likewise been employed for a plethora of diagnostic purposes. Nonetheless, multiplexed PCR-based detection schemes have hitherto been largely limited by technical challenges associated with nonspecific interactions and other limitations inherent to traditional fluorescence-based assays. Here, we describe a novel strategy for multiplexed PCR-based analysis called Ligation-eNabled fluorescence-Coding PCR (LiNC PCR) that exponentially enhances the multiplexing capability of standard fluorescence-based PCR assays. The technique relies upon a simple, preliminary ligation reaction in which target DNA sequences are converted to PCR template molecules with distinct endpoint fluorescence signatures. Universal TaqMan probes are used to create target-specific multicolor fluorescence signals that can be readily decoded to identify amplified targets of interest. We demonstrate the LiNC PCR technique by implementing a two-color-based assay for detection of 10 ovarian cancer epigenetic biomarkers at analytical sensitivities as low as 60 template molecules with no detectable target cross-talk. Overall, LiNC PCR provides a simple and inexpensive method for achieving high-dimensional multiplexing that can be implemented in manifold molecular diagnostic applications.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN54</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Park, Joon Soo and Pisanic, Thomas and Zhang, Ye and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ligation-Enabled Fluorescence-Coding PCR for High-Dimensional Fluorescence-Based Nucleic Acid Detection}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Analytical Chemistry}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0003-2700}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/acs.analchem.0c04221}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/acs.analchem.0c04221}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">J. Pathol.</abbr></div> <div id="RN67" class="col-sm-9"> <div class="title">Mutation and methylation profiles of ectopic and eutopic endometrial tissues</div> <div class="author">Li, Lihong, Antero, Maria Facadio, Zhang, Ming, Chu, Tiffany, Seckin, Tamer, Ayhan, Ayse,  Pisanic, Thomas, Wang, Tian-Li, Cope, Leslie, Segars, James, and Shih, Ie-Ming </div> <div class="periodical"> <em>The Journal of Pathology</em>, 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.5778" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Abstract Adenomyosis and peritoneal endometriosis are common gynecologic lesions; they are characterized by aberrant locations of normal-appearing endometrium in myometrium and peritoneal surface, respectively. Both ectopic lesions are speculated to originate from uterine eutopic endometrium, which is composed of epithelium and stroma, but how these two different tissue types co-evolve in ectopic locations remains unclear. Here, we analyzed exome-wide mutations and global methylation in microdissected epithelium and stroma separately in paired adenomyosis, peritoneal endometriosis, and endometrium to investigate their relationship. Analyses of somatic mutations and their allele frequencies indicate monoclonal development not only in epithelium but also in the stroma of adenomyosis and peritoneal endometriosis. Our preliminary phylogenetic study suggests a plausible clonal derivation in epithelium and stroma of both ectopic and eutopic endometrium from the same founder epithelium-stroma progenitor cells. While a patient-specific methylation landscape is evident, adenomyosis epithelium and stroma can be distinguished from normal-appearing eutopic endometrium epigenetically. In summary, endometrial stroma, like its epithelial counterpart, could be clonal and both ectopic and eutopic endometrium following divergent evolutionary trajectories. Our data also warrant future investigations into the role of endometrial stroma in the pathobiology of endometrium-related disorders. © 2021 The Pathological Society of Great Britain and Ireland. Published by John Wiley &amp; Sons, Ltd.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN67</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Lihong and Antero, Maria Facadio and Zhang, Ming and Chu, Tiffany and Seckin, Tamer and Ayhan, Ayse and Pisanic, Thomas and Wang, Tian-Li and Cope, Leslie and Segars, James and Shih, Ie-Ming}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Mutation and methylation profiles of ectopic and eutopic endometrial tissues}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The Journal of Pathology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{255}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{387-398}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0022-3417}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1002/path.5778}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/path.5778}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Epigenetics</abbr></div> <div id="RN69" class="col-sm-9"> <div class="title">High performance methylated DNA markers for detection of colon adenocarcinoma</div> <div class="author">Klein Kranenbarg, Romy A. M., Vali, Abdul Hussain, Ijzermans, Jan N. M.,  Pisanic, Thomas R., Wang, Tza-Huei, Azad, Nilofer, Sukumar, Saraswati, and Fackler, Mary Jo </div> <div class="periodical"> <em>Clinical Epigenetics</em>, 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13148-021-01206-2" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Colon cancer (CC) is treatable if detected in its early stages. Improved CC detection assays that are highly sensitive, specific, and available at point of care are needed. In this study, we systematically selected and tested methylated markers that demonstrate high sensitivity and specificity for detection of CC in tissue and circulating cell-free DNA.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN69</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Klein Kranenbarg, Romy A. M. and Vali, Abdul Hussain and Ijzermans, Jan N. M. and Pisanic, Thomas R. and Wang, Tza-Huei and Azad, Nilofer and Sukumar, Saraswati and Fackler, Mary Jo}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High performance methylated DNA markers for detection of colon adenocarcinoma}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Epigenetics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{218}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1868-7083}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13148-021-01206-2}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13148-021-01206-2}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Transl. Sci.</abbr></div> <div id="RN55" class="col-sm-9"> <div class="title">A phase 2 trial of gemcitabine and docetaxel in patients with metastatic colorectal adenocarcinoma with methylated checkpoint with forkhead and ring finger domain promoter and/or microsatellite instability phenotype</div> <div class="author">Baretti, Marina, Karunasena, Enusha, Zahurak, Marianna, Walker, Rosalind, Zhao, Yang,  Pisanic, Thomas R., Wang, Tza-Huei, Greten, Tim F., Duffy, Austin G., Gootjes, Elske, Meijer, Gerrit, Verheul, Henk M.W., Ahuja, Nita, Herman, James G., and Azad, Nilofer S. </div> <div class="periodical"> <em>Clinical and Translational Science</em>, 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ascpt.onlinelibrary.wiley.com/doi/abs/10.1111/cts.12960" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Abstract We previously reported CHFR methylation in a subset of colorectal cancer (CRC; ∼30%) with high concordance with microsatellite instability (MSI). We also showed that CHFR methylation predicted for sensitivity to docetaxel, whereas the MSI-high phenotypes were sensitive to gemcitabine. We hypothesized that this subset of patients with CRC would be selectively sensitive to gemcitabine and docetaxel. We enrolled a Phase 2 trial of gemcitabine and docetaxel in patients with MSI-high and/or CHFR methylated CRC. The primary objective was Response Evaluation Criteria in Solid Tumors (RECIST) 1.1 response rate. Enrolled patients were treated with gemcitabine 800 mg/m2 on days 1 and 8 and docetaxel 70 mg/m2 on day 8 of each 21-day cycle. A total of 6 patients with CHFR-methylated, MSI-high CRC were enrolled from September 2012 to August 2016. The study was closed in September of 2017 due to poor accrual prior to reaching the first interim assessment of response rate, which would have occurred at 10 patients. No RECIST criteria tumor responses were observed, with 3 patients (50%) having stable disease as best response, 1 lasting more than 9 months. Median progression-free survival (PFS) was 1.79 months (95% confidence interval [CI] = 1.28, not available [NA]) and median overall survival (OS) was 15.67 months (95% CI = 4.24, NA). Common grade 3 toxicities were lymphopenia (67%), leukopenia (33%), and anemia (33%). Although negative, this study establishes a proof-of-concept for the implementation of epigenetic biomarkers (CHFR methylation/MSI) as inclusion criteria in a prospective clinical trial to optimize combinatorial strategies in the era of personalized medicine.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN55</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Baretti, Marina and Karunasena, Enusha and Zahurak, Marianna and Walker, Rosalind and Zhao, Yang and Pisanic, Thomas R. and Wang, Tza-Huei and Greten, Tim F. and Duffy, Austin G. and Gootjes, Elske and Meijer, Gerrit and Verheul, Henk M.W. and Ahuja, Nita and Herman, James G. and Azad, Nilofer S.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A phase 2 trial of gemcitabine and docetaxel in patients with metastatic colorectal adenocarcinoma with methylated checkpoint with forkhead and ring finger domain promoter and/or microsatellite instability phenotype}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical and Translational Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1752-8054}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1111/cts.12960}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://ascpt.onlinelibrary.wiley.com/doi/abs/10.1111/cts.12960}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Epigenetics</abbr></div> <div id="RN44" class="col-sm-9"> <div class="title">Leveraging locus-specific epigenetic heterogeneity to improve the performance of blood-based DNA methylation biomarkers</div> <div class="author">Miller, Brendan F.,  Pisanic, Thomas R., Margolin, Gennady, Petrykowska, Hanna M., Athamanolap, Pornpat, Osei-Tutu, Akosua, Annunziata, Christina M., Wang, Tza-Huei, and Elnitski, Laura </div> <div class="periodical"> <em>Clinical Epigenetics</em>, 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-020-00939-w" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Aberrant DNA methylation is commonly heralded as a promising cancer biomarker; however, its inherently stochastic nature often leads to variable methylation patterns that can complicate the use of methylation biomarkers for clinical diagnostics, particularly in dilute samples such as liquid biopsies. Here, we present a methylation density binary classifier, a statistical method for leveraging differential heterogeneous methylation to predict and optimize the performance of methylation biomarkers for clinical applications. We first developed and tested the classifier using methylation density profiles derived from reduced representation bisulfite sequencing reads of ovarian carcinoma at ZNF154, a recurrently methylated locus in multiple cancer types. We then used in silico simulations to predict the performance of the classifier in liquid biopsies and validated these predictions using quasi-digital melt curve analysis (DREAMing) of circulating cell-free DNA from individuals with versus without ovarian carcinoma. We found good agreement between predicted and observed classifier performance, and further demonstrated that implementation of this approach with ZNF154 outperformed CA-125 for use in etiologically-diverse ovarian cancer types. Our results indicate that methylation density profiles can be exploited to predict and facilitate implementation of methylation biomarkers for clinical applications, and that ZNF154 methylation shows promise as a clinically-useful biomarker for ovarian cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN44</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Miller, Brendan F. and Pisanic, Thomas R. and Margolin, Gennady and Petrykowska, Hanna M. and Athamanolap, Pornpat and Osei-Tutu, Akosua and Annunziata, Christina M. and Wang, Tza-Huei and Elnitski, Laura}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Leveraging locus-specific epigenetic heterogeneity to improve the performance of blood-based DNA methylation biomarkers}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Epigenetics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13148-020-00939-w}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/10.1101/579839v2}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Cancer Res.</abbr></div> <div id="RN50" class="col-sm-9"> <div class="title">Methylomic Landscapes of Ovarian Cancer Precursor Lesions</div> <div class="author"> Pisanic, Thomas R., Wang, Yeh, Sun, Hanru, Considine, Michael, Li, Lihong, Wang, Tza-Huei, Wang, Tian-Li, and Shih, Ie-Ming </div> <div class="periodical"> <em>Clinical Cancer Research</em>, 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://aacrjournals.org/clincancerres/article/26/23/6310/83008/Methylomic-Landscapes-of-Ovarian-Cancer-Precursor" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Purpose: The current paradigm in the development of high-grade serous ovarian carcinoma (HGSC) proposes that the majority of HGSCs arise from precursor serous tubal intraepithelial carcinoma (STIC) lesions of the fallopian tube. Here we survey genome-wide methylation in HGSC precursor lesions to identify genomic regions that exhibit high-specificity differential hypermethylation for potential use as biomarkers for detecting STIC and HGSC at stages when curative intervention likely remains feasible.Experimental Design: We first identified quality control criteria for performing reliable methylomic analysis of DNA-limited tubal precursor lesions with the Illumina Infinium MethylationEPIC array. We then used this platform to compare genome-wide methylation among 12 STICs with paired adjacent-normal epithelia, one p53 signature lesion and two samples of concurrent HGSC. The resulting methylomic data were analyzed by unsupervised hierarchical clustering and multidimensional analysis. Regions of high-confidence STIC-specific differential hypermethylation were identified using selective bioinformatic criteria and compared with published MethylationEPIC data from 23 HGSC tumors and 11 healthy fallopian tube mucosae.Results: Unsupervised analysis showed that STICs largely clustered with HGSCs, but were clearly distinct from adjacent-normal fallopian tube epithelia. Forty-two genomic regions exhibited high-confidence STIC-specific differential hypermethylation, of which 17 (40.5%) directly overlapped with HGSC-specific differentially methylated regions. Methylation at these shared loci was able to completely distinguish STIC and HGSC samples from normal and adjacent-normal specimens.Conclusions: Our results suggest that most STICs are epigenetically similar to HGSCs and share regions of differential hypermethylation that warrant further evaluation for potential use as biomarkers for early detection of ovarian HGSC.See related commentary by Ishak and De Carvalho, p. 6083</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN50</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, Thomas R. and Wang, Yeh and Sun, Hanru and Considine, Michael and Li, Lihong and Wang, Tza-Huei and Wang, Tian-Li and Shih, Ie-Ming}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Methylomic Landscapes of Ovarian Cancer Precursor Lesions}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Cancer Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{26}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{23}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{6310-6320}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1158/1078-0432.Ccr-20-0270}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cell Rep</abbr></div> <div id="RN53" class="col-sm-9"> <div class="title">Epigenomic Reprogramming toward Mesenchymal-Epithelial Transition in Ovarian-Cancer-Associated Mesenchymal Stem Cells Drives Metastasis</div> <div class="author">Fan, Huihui, Atiya, Huda I., Wang, Yeh,  Pisanic, Thomas R., Wang, Tza-Huei, Shih, Ie-Ming, Foy, Kelly K., Frisbie, Leonard, Buckanovich, Ronald J., Chomiak, Alison A., Tiedemann, Rochelle L., Rothbart, Scott B., Chandler, Chelsea, Shen, Hui, and Coffman, Lan G. </div> <div class="periodical"> <em>Cell Rep.</em>, 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1016/j.celrep.2020.108473" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>A role for cancer cell epithelial-to-mesenchymal transition (EMT) in cancer is well established. Here, we show that, in addition to cancer cell EMT, ovarian cancer cell metastasis relies on an epigenomic mesenchymal-to-epithelial transition (MET) in host mesenchymal stem cells (MSCs). These reprogrammed MSCs, termed carcinoma-associated MSCs (CA-MSCs), acquire pro-tumorigenic functions and directly bind cancer cells to serve as a metastatic driver/chaperone. Cancer cells induce this epigenomic MET characterized by enhancer-enriched DNA hypermethylation, altered chromatin accessibility, and differential histone modifications. This phenomenon appears clinically relevant, as CA-MSC MET is highly correlated with patient survival. Mechanistically, mirroring MET observed in development, MET in CA-MSCs is mediated by WT1 and EZH2. Importantly, EZH2 inhibitors, which are clinically available, significantly inhibited CA-MSC-mediated metastasis in mouse models of ovarian cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN53</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Fan, Huihui and Atiya, Huda I. and Wang, Yeh and Pisanic, Thomas R. and Wang, Tza-Huei and Shih, Ie-Ming and Foy, Kelly K. and Frisbie, Leonard and Buckanovich, Ronald J. and Chomiak, Alison A. and Tiedemann, Rochelle L. and Rothbart, Scott B. and Chandler, Chelsea and Shen, Hui and Coffman, Lan G.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Epigenomic Reprogramming toward Mesenchymal-Epithelial Transition in Ovarian-Cancer-Associated Mesenchymal Stem Cells Drives Metastasis}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Rep.}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{33}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2211-1247}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.celrep.2020.108473}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.celrep.2020.108473}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Am. J. Clin. Pathol.</abbr></div> <div id="RN37" class="col-sm-9"> <div class="title">Long Interspersed Nuclear Element 1 Retrotransposons Become Deregulated during the Development of Ovarian Cancer Precursor Lesions</div> <div class="author"> Pisanic, Thomas R., Asaka, Shiho, Lin, Shiou-Fu, Yen, Ting-Tai, Sun, Hanru, Bahadirli-Talbott, Asli, Wang, Tza-Huei, Burns, Kathleen H., Wang, Tian-Li, and Shih, Ie-Ming </div> <div class="periodical"> <em>The American Journal of Pathology</em>, 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://www.sciencedirect.com/science/article/pii/S0002944018306904" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>There is growing evidence that most high-grade serous ovarian carcinomas likely arise from local dissemination of precursor lesions of the fallopian tube. Evolution of these lesions from early p53 signatures to latter-stage, serous tubal intraepithelial carcinomas (STICs) is characterized by cytologic atypia, accumulation of somatic mutations, and genomic instability, the etiologies of which remain unclear. Long interspersed element 1 (LINE-1) retrotransposon is expressed in many carcinomas, including high-grade serous ovarian carcinoma, where it contributes to genomic instability; however, the timing of LINE-1 activation during this evolution has yet to be elucidated. In this study, we assessed LINE-1 open reading frame 1 protein expression in 12 p53 signature lesions, 32 STICs, and 112 various types of ovarian cancers via immunohistochemical staining and examined LINE-1 promoter methylation in representative cases. We found that 78% and 57% of STICs, with and without concurrent ovarian carcinomas, respectively, exhibited intense LINE-1 immunoreactivity compared with adjacent, normal-appearing fallopian tube epithelium. Hypomethylation of the LINE-1 promoter was found in all STICs exhibiting overexpression. None of the 12 p53 signatures demonstrated significant LINE-1 expression. In ovarian cancer, 84 (75%) of 112 ovarian carcinomas overexpressed LINE-1. Our results indicate that LINE-1 retrotransposons often become deregulated during progression of ovarian cancer precursor lesions from the p53 signature to STIC stages and remain highly expressed in carcinoma.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN37</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, Thomas R. and Asaka, Shiho and Lin, Shiou-Fu and Yen, Ting-Tai and Sun, Hanru and Bahadirli-Talbott, Asli and Wang, Tza-Huei and Burns, Kathleen H. and Wang, Tian-Li and Shih, Ie-Ming}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Long Interspersed Nuclear Element 1 Retrotransposons Become Deregulated during the Development of Ovarian Cancer Precursor Lesions}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The American Journal of Pathology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{189}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{513-520}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0002-9440}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.ajpath.2018.11.005}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.sciencedirect.com/science/article/pii/S0002944018306904}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Lab Chip</abbr></div> <div id="RN28" class="col-sm-9"> <div class="title">Multilayer microfluidic array for highly efficient sample loading and digital melt analysis of DNA methylation</div> <div class="author">O’Keefe, Christine M., Giammanco, Daniel, Li, Sixuan,  Pisanic, Thomas R., and Wang, Tza-Huei Jeff </div> <div class="periodical"> <em>Lab on a Chip</em>, 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1039/C8LC01189C" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Liquid biopsies contain a treasure of genetic and epigenetic biomarkers that contain information for the detection and monitoring of human disease. DNA methylation is an epigenetic modification that is critical to determining cellular phenotype and often becomes altered in many disease states. In cancer, aberrant DNA methylation contributes to carcinogenesis and can profoundly affect tumor evolution, metastatic potential, and resistance to therapeutic intervention. However, current technologies are not well-suited for quantitative assessment of DNA methylation heterogeneity, especially in challenging samples such as liquid biopsies with low DNA input and high background. We present a multilayer microfluidic device for quantitative analysis of DNA methylation by digital PCR and high resolution melt (HRM). The multilayer design facilitates high-density array digitization aimed at maximizing sample loading efficiency. The platform achieves highly parallelized digital PCR-HRM-based discrimination of rare heterogeneous DNA methylation as low as 0.0001% methylated/unmethylated molecules of a classic tumor suppressor gene, CDKN2A (p14ARF).</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN28</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{O'Keefe, Christine M. and Giammanco, Daniel and Li, Sixuan and Pisanic, Thomas R. and Wang, Tza-Huei Jeff}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multilayer microfluidic array for highly efficient sample loading and digital melt analysis of DNA methylation}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Lab on a Chip}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{19}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{444-451}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1473-0197}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1039/C8LC01189C}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1039/C8LC01189C}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Epigenetics</abbr></div> <div id="RN38" class="col-sm-9"> <div class="title">Promoter methylation of ADAMTS1 and BNC1 as potential biomarkers for early detection of pancreatic cancer in blood</div> <div class="author">Eissa, Maryam A. L., Lerner, Lane, Abdelfatah, Eihab, Shankar, Nakul, Canner, Joseph K., Hasan, Nesrin M., Yaghoobi, Vesal, Huang, Barry, Kerner, Zachary, Takaesu, Felipe, Wolfgang, Christopher, Kwak, Ruby, Ruiz, Michael, Tam, Matthew,  Pisanic, Thomas R., Iacobuzio-Donahue, Christine A., Hruban, Ralph H., He, Jin, Wang, Tza-Huei, Wood, Laura D., Sharma, Anup, and Ahuja, Nita </div> <div class="periodical"> <em>Clinical Epigenetics</em>, 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13148-019-0650-0" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Despite improvements in cancer management, most pancreatic cancers are still diagnosed at an advanced stage. We have recently identified promoter DNA methylation of the genes ADAMTS1 and BNC1 as potential blood biomarkers of pancreas cancer. In this study, we validate this biomarker panel in peripheral cell-free tumor DNA of patients with pancreatic cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN38</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Eissa, Maryam A. L. and Lerner, Lane and Abdelfatah, Eihab and Shankar, Nakul and Canner, Joseph K. and Hasan, Nesrin M. and Yaghoobi, Vesal and Huang, Barry and Kerner, Zachary and Takaesu, Felipe and Wolfgang, Christopher and Kwak, Ruby and Ruiz, Michael and Tam, Matthew and Pisanic, Thomas R. and Iacobuzio-Donahue, Christine A. and Hruban, Ralph H. and He, Jin and Wang, Tza-Huei and Wood, Laura D. and Sharma, Anup and Ahuja, Nita}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Promoter methylation of ADAMTS1 and BNC1 as potential biomarkers for early detection of pancreatic cancer in blood}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Epigenetics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{59}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1868-7083}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13148-019-0650-0}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13148-019-0650-0}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">EBioMedicine</abbr></div> <div id="RN41" class="col-sm-9"> <div class="title">Rab8 GTPase regulates Klotho-mediated inhibition of cell growth and progression by directly modulating its surface expression in human non-small cell lung cancer</div> <div class="author">Chen, Bo, Huang, Shuhong,  Pisanic, Thomas R., Stark, Alejandro, Tao, Yong, Cheng, Bei, Li, Yue, Wei, Yunyan, Zhao, Weihong, Wang, Tza-Huei, and Wu, Jianqing </div> <div class="periodical"> <em>EBioMedicine</em>, 2019 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1016/j.ebiom.2019.10.040" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN41</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chen, Bo and Huang, Shuhong and Pisanic, Thomas R. and Stark, Alejandro and Tao, Yong and Cheng, Bei and Li, Yue and Wei, Yunyan and Zhao, Weihong and Wang, Tza-Huei and Wu, Jianqing}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Rab8 GTPase regulates Klotho-mediated inhibition of cell growth and progression by directly modulating its surface expression in human non-small cell lung cancer}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{EBioMedicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{49}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{118-132}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2352-3964}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.ebiom.2019.10.040}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.ebiom.2019.10.040}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Cancer Res.</abbr></div> <div id="RN30" class="col-sm-9"> <div class="title">Methylomic Analysis of Ovarian Cancers Identifies Tumor-Specific Alterations Readily Detectable in Early Precursor Lesions</div> <div class="author"> Pisanic, Thomas R., Cope, Leslie M., Lin, Shiou-Fu, Yen, Ting-Tai, Athamanolap, Pornpat, Asaka, Ryoichi, Nakayama, Kentaro, Fader, Amanda N., Wang, Tza-Huei, Shih, Ie-Ming, and Wang, Tian-Li </div> <div class="periodical"> <em>Clinical Cancer Research</em>, 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://clincancerres.aacrjournals.org/content/clincanres/24/24/6536.full.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Purpose: High-grade serous ovarian carcinoma (HGSOC) typically remains undiagnosed until advanced stages when peritoneal dissemination has already occurred. Here, we sought to identify HGSOC-specific alterations in DNA methylation and assess their potential to provide sensitive and specific detection of HGSOC at its earliest stages.Experimental Design: MethylationEPIC genome-wide methylation analysis was performed on a discovery cohort comprising 23 HGSOC, 37 non-HGSOC malignant, and 36 histologically unremarkable gynecologic tissue samples. The resulting data were processed using selective bioinformatic criteria to identify regions of high-confidence HGSOC-specific differential methylation. Quantitative methylation-specific real-time PCR (qMSP) assays were then developed for 8 of the top-performing regions and analytically validated in a cohort of 90 tissue samples. Lastly, qMSP assays were used to assess and compare methylation in 30 laser-capture microdissected (LCM) fallopian tube epithelia samples obtained from cancer-free and serous tubal intraepithelial carcinoma (STIC) positive women.Results: Bioinformatic selection identified 91 regions of robust, HGSOC-specific hypermethylation, 23 of which exhibited an area under the receiver-operator curve (AUC) value ≥ 0.9 in the discovery cohort. Seven of 8 top-performing regions demonstrated AUC values between 0.838 and 0.968 when analytically validated by qMSP in a 90-patient cohort. A panel of the 3 top-performing genes (c17orf64, IRX2, and TUBB6) was able to perfectly discriminate HGSOC (AUC 1.0). Hypermethylation within these loci was found exclusively in LCM fallopian tube epithelia from women with STIC lesions, but not in cancer-free fallopian tubes.Conclusions: A panel of methylation biomarkers can be used to accurately identify HGSOC, even at precursor stages of the disease.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN30</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, Thomas R. and Cope, Leslie M. and Lin, Shiou-Fu and Yen, Ting-Tai and Athamanolap, Pornpat and Asaka, Ryoichi and Nakayama, Kentaro and Fader, Amanda N. and Wang, Tza-Huei and Shih, Ie-Ming and Wang, Tian-Li}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Methylomic Analysis of Ovarian Cancers Identifies Tumor-Specific Alterations Readily Detectable in Early Precursor Lesions}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Cancer Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{24}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{24}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{6536-6547}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1158/1078-0432.Ccr-18-1199}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Ann. Oncol.</abbr></div> <div id="RN16" class="col-sm-9"> <div class="title">Phase II study of nab-paclitaxel in refractory small bowel adenocarcinoma and CpG island methylator phenotype (CIMP)-high colorectal cancer</div> <div class="author">Overman, M. J., Adam, L., Raghav, K., Wang, J., Kee, B., Fogelman, D., Eng, C., Vilar, E., Shroff, R., Dasari, A., Wolff, R., Morris, J., Karunasena, E.,  Pisanic, T., Azad, N., and Kopetz, S. </div> <div class="periodical"> <em>Annals of Oncology</em>, 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1093/annonc/mdx688" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>BackgroundHypermethylation of promoter CpG islands [CpG island methylator phenotype (CIMP)] represents a unique pathway for the development of colorectal cancer (CRC), characterized by lack of chromosomal instability and a low rate of adenomatous polyposis coli (APC) mutations, which have both been correlated with taxane resistance. Similarly, small bowel adenocarcinoma (SBA), a rare tumor, also has a low rate of APC mutations. This phase II study evaluated taxane sensitivity in SBA and CIMP-high CRC.Patients and methodsThe primary objective was Response Evaluation Criteria in Solid Tumors version 1.1 response rate. Eligibility included Eastern Cooperative Oncology Group performance status 0/1, refractory disease, and SBA or CIMP-high metastatic CRC. Nab-paclitaxel was initially administered at a dose of 260 mg/m2 every 3 weeks but was reduced to 220 mg/m2 owing to toxicity.ResultsA total of 21 patients with CIMP-high CRC and 13 with SBA were enrolled from November 2012 to October 2014. The efficacy-assessable population (patients who received at least three doses of the treatment) comprised 15 CIMP-high CRC patients and 10 SBA patients. Common grade 3 or 4 toxicities were fatigue (12%), neutropenia (9%), febrile neutropenia (9%), dehydration (6%), and thrombocytopenia (6%). No responses were seen in the CIMP-high CRC cohort and two partial responses were seen in the SBA cohort. Median progression-free survival was significantly greater in the SBA cohort than in the CIMP-high CRC cohort (3.2 months compared with 2.1 months, P = 0.03). Neither APC mutation status nor CHFR methylation status correlated with efficacy in the CIMP-high CRC cohort. In vivo testing of paclitaxel in an SBA patient-derived xenograft validated the activity of taxanes in this disease type.ConclusionAlthough preclinical studies suggested taxane sensitivity was associated with chromosomal stability and wild-type APC, we found that nab-paclitaxel was inactive in CIMP-high metastatic CRC. Nab-paclitaxel may represent a novel therapeutic option for SBA.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN16</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Overman, M. J. and Adam, L. and Raghav, K. and Wang, J. and Kee, B. and Fogelman, D. and Eng, C. and Vilar, E. and Shroff, R. and Dasari, A. and Wolff, R. and Morris, J. and Karunasena, E. and Pisanic, T. and Azad, N. and Kopetz, S.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Phase II study of nab-paclitaxel in refractory small bowel adenocarcinoma and CpG island methylator phenotype (CIMP)-high colorectal cancer}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Annals of Oncology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{29}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{139-144}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0923-7534}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/annonc/mdx688}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1093/annonc/mdx688}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Sci. Adv.</abbr></div> <div id="RN39" class="col-sm-9"> <div class="title">Facile profiling of molecular heterogeneity by microfluidic digital melt</div> <div class="author">O’Keefe, Christine M.,  Pisanic, Thomas R., Zec, Helena, Overman, Michael J., Herman, James G., and Wang, Tza-Huei </div> <div class="periodical"> <em>Science Advances</em>, 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/10.1126/sciadv.aat6459" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>This work presents a digital microfluidic platform called HYPER-Melt (high-density profiling and enumeration by melt) for highly parallelized copy-by-copy DNA molecular profiling. HYPER-Melt provides a facile means of detecting and assessing sequence variations of thousands of individual DNA molecules through digitization in a nanowell microchip array, allowing amplification and interrogation of individual template molecules by detecting HRM fluorescence changes due to sequence-dependent denaturation. As a model application, HYPER-Melt is used here for the detection and assessment of intermolecular heterogeneity of DNA methylation within the promoters of classical tumor suppressor genes. The capabilities of this platform are validated through serial dilutions of mixed epialleles, with demonstrated detection limits as low as 1 methylated variant in 2 million unmethylated templates (0.00005%) of a classic tumor suppressor gene, CDKN2A (p14ARF). The clinical potential of the platform is demonstrated using a digital assay for NDRG4, a tumor suppressor gene that is commonly methylated in colorectal cancer, in liquid biopsies of healthy and colorectal cancer patients. Overall, the platform provides the depth of information, simplicity of use, and single-molecule sensitivity necessary for rapid assessment of intermolecular variation contributing to genetic and epigenetic heterogeneity for challenging applications in embryogenesis, carcinogenesis, and rare biomarker detection.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN39</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{O’Keefe, Christine M. and Pisanic, Thomas R. and Zec, Helena and Overman, Michael J. and Herman, James G. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Facile profiling of molecular heterogeneity by microfluidic digital melt}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science Advances}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{eaat6459}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/sciadv.aat6459}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge"> Semin. Cell Dev. Biol.</abbr></div> <div id="RN1" class="col-sm-9"> <div class="title">Defining, distinguishing and detecting the contribution of heterogeneous methylation to cancer heterogeneity</div> <div class="author"> Pisanic, T. R., Athamanolap, P., and Wang, T. H. </div> <div class="periodical"> <em>Seminars in Cell &amp; Developmental Biology</em>, 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="&lt;Go%20to%20ISI&gt;://WOS:000398612700002" class="btn btn-sm z-depth-0" role="button">HTML</a> </div> <div class="abstract hidden"> <p>DNA methylation is a fundamental means of epigenetic gene regulation that occurs in virtually all cell types. In many higher organisms, including humans, it plays vital roles in cell differentiation and homeostatic maintenance of cell phenotype. The control of DNA methylation has traditionally been attributed to a highly coordinated, linear process, whose dysregulation has been associated with numerous pathologies including cancer, where it occurs early in, and even prior to, the development of neoplastic tissues. Recent experimental evidence has demonstrated that, contrary to prevailing paradigms, methylation patterns are actually maintained through inexact, dynamic processes. These processes normally result in minor stochastic differences between cells that accumulate with age. However, various factors, including cancer itself, can lead to substantial differences in intercellular methylation patterns, viz. methylation heterogeneity. Advancements in molecular biology techniques are just now beginning to allow insight into how this heterogeneity contributes to clonal evolution and overall cancer heterogeneity. In the current review, we begin by presenting a didactic overview of how the basal bimodal methylome is established and maintained. We then provide a synopsis of some of the factors that lead to the accrual of heterogeneous methylation and how this heterogeneity may lead to gene silencing and impact the development of cancerous phenotypes. Lastly, we highlight currently available methylation assessment techniques and discuss their suitability to the study of heterogeneous methylation. (C) 2016 Published by Elsevier Ltd.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN1</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, T. R. and Athamanolap, P. and Wang, T. H.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Defining, distinguishing and detecting the contribution of heterogeneous methylation to cancer heterogeneity}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Seminars in Cell &amp; Developmental Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{64}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{5-17}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1084-9521}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.semcdb.2016.08.030}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{&lt;Go to ISI&gt;://WOS:000398612700002}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cancer Res.</abbr></div> <div id="RN33" class="col-sm-9"> <div class="title">Abstract 4666: DREAMing as a simple and low cost alternative for the assessment of methylation in ultra rare DNA</div> <div class="author"> Pisanic, Thomas R., Athamanolap, Pornpat, Miller, Brendan F., Wu, Vincent, Elnitski, Laura, and Wang, Tza-Huei </div> <div class="periodical"> <em>Cancer Research</em>, 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Proceedings: AACR Annual Meeting 2017; April 1-5, 2017; Washington, DCBackground: Current approaches for the assessment of methylation, such as methylation-specific PCR (MSP) and next-generation bisulfite sequencing (BS-Seq) are fundamentally limited in their ability to detect and assess heterogeneous methylation patterns (epialleles) in ultra-rare (&lt;0.1%) DNA. These limitations critically compromise diagnostic utility and render them ill suited for many emerging applications in cancer diagnostics, such as the analysis of methylation heterogeneity in cell-free DNA (cfDNA) and rare cell populations. We recently addressed the need for a low cost alternative to the assessment of methylation of ultra-rare DNA with the development of DREAMing (Discrimination of Rare EpiAlleles by Melt), which uses semi-limiting dilution and precise melt curve analysis to distinguish and enumerate individual copies of DNA at single copy sensitivity and single-CpG-site resolution. Here, we seek to demonstrate the advantages of the DREAMing method over conventional approaches to methylation assessment.Methods: We expand upon the underlying theory of DREAMing and provide guidelines for the development of single-copy sensitive DREAMing assays. We further elucidate methods for tailoring DREAMing assays to samples of interest and compare the performance of these assays to commonly employed techniques including quantitative MSP (qMSP) and BS-Seq.Results: Development of single-copy sensitive DREAMing assays for a number of loci associated with classic tumor-specific methylation such as CHFR and RASSF1A as well as a candidate pan-cancer locus are reported. These assays are then used to analyze methylation in cfDNA derived from the plasma of cancer-positive and healthy patients. DREAM analysis reveals that DREAMing can readily detect over an order of magnitude more epialleles when directly compared to qMSP and BS-Seq assays of the same locus. Some of the challenges associated with distinguishing potential tumor-specific aberrant methylation from background methylation are then discussed and proposed solutions are demonstrated. Lastly, methods for optimizing DREAMing assays for specific sample types are discussed.Conclusions: DREAMing is a recently introduced method for the assessment of locus-specific methylation in samples containing ultra rare target DNA. Its low cost and simplicity coupled with the ability to provide enhanced, single-copy detection of heterogeneous methylation make DREAMing an attractive option over traditional techniques for demanding specimens such as cfDNA and rare cell populations. DREAMing has potential utility in the evaluation of DNA methylation dynamics in cell populations, prenatal testing, as well as clear use in early cancer diagnostic, companion diagnostic and predictive applications.Citation Format: Thomas R. Pisanic, Pornpat Athamanolap, Brendan F. Miller, Vincent Wu, Laura Elnitski, Tza-Huei Wang. DREAMing as a simple and low cost alternative for the assessment of methylation in ultra rare DNA [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2017; 2017 Apr 1-5; Washington, DC. Philadelphia (PA): AACR; Cancer Res 2017;77(13 Suppl):Abstract nr 4666. doi:10.1158/1538-7445.AM2017-4666</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN33</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, Thomas R. and Athamanolap, Pornpat and Miller, Brendan F. and Wu, Vincent and Elnitski, Laura and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Abstract 4666: DREAMing as a simple and low cost alternative for the assessment of methylation in ultra rare DNA}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cancer Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{77}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{13 Supplement}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{4666-4666}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1158/1538-7445.Am2017-4666}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2016</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Biomed. Microdevices</abbr></div> <div id="RN34" class="col-sm-9"> <div class="title">A parallelized microfluidic DNA bisulfite conversion module for streamlined methylation analysis</div> <div class="author">Stark, Alejandro, Shin, Dong Jin,  Pisanic, Thomas, Hsieh, Kuangwen, and Wang, Tza-Huei </div> <div class="periodical"> <em>Biomedical Microdevices</em>, 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1007/s10544-015-0029-8" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Aberrant methylation of DNA has been identified as an epigenetic biomarker for numerous cancer types. The vast majority of techniques aimed at detecting methylation require bisulfite conversion of the DNA sample prior to analysis, which until now has been a benchtop process. Although microfluidics has potential benefits of simplified operation, sample and reagent economy, and scalability, bisulfite conversion has yet to be implemented in this format. Here, we present a novel droplet microfluidic design that facilitates rapid bisulfite conversion by reducing the necessary processing steps while retaining comparable performance to existing methods. This new format has a reduced overall processing time and is readily scalable for use in high throughput DNA methylation analysis.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN34</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Stark, Alejandro and Shin, Dong Jin and Pisanic, Thomas and Hsieh, Kuangwen and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A parallelized microfluidic DNA bisulfite conversion module for streamlined methylation analysis}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biomedical Microdevices}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{18}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1572-8781}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10544-015-0029-8}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1007/s10544-015-0029-8}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="year">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nucleic Acids Res.</abbr></div> <div id="RN75" class="col-sm-9"> <div class="title">DREAMing: a simple and ultrasensitive method for assessing intratumor epigenetic heterogeneity directly from liquid biopsies</div> <div class="author"> Pisanic II, Thomas R., Athamanolap, Pornpat, Poh, Weijie, Chen, Chen, Hulbert, Alicia, Brock, Malcolm V., Herman, James G., and Wang, Tza-Huei </div> <div class="periodical"> <em>Nucleic Acids Research</em>, 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1093/nar/gkv795" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Many cancers comprise heterogeneous populations of cells at primary and metastatic sites throughout the body. The presence or emergence of distinct subclones with drug-resistant genetic and epigenetic phenotypes within these populations can greatly complicate therapeutic intervention. Liquid biopsies of peripheral blood from cancer patients have been suggested as an ideal means of sampling intratumor genetic and epigenetic heterogeneity for diagnostics, monitoring and therapeutic guidance. However, current molecular diagnostic and sequencing methods are not well suited to the routine assessment of epigenetic heterogeneity in difficult samples such as liquid biopsies that contain intrinsically low fractional concentrations of circulating tumor DNA (ctDNA) and rare epigenetic subclonal populations. Here we report an alternative approach, deemed DREAMing (Discrimination of Rare EpiAlleles by Melt), which uses semi-limiting dilution and precise melt curve analysis to distinguish and enumerate individual copies of epiallelic species at single-CpG-site resolution in fractions as low as 0.005%, providing facile and inexpensive ultrasensitive assessment of locus-specific epigenetic heterogeneity directly from liquid biopsies. The technique is demonstrated here for the evaluation of epigenetic heterogeneity at p14ARF and BRCA1 gene-promoter loci in liquid biopsies obtained from patients in association with non-small cell lung cancer (NSCLC) and myelodysplastic/myeloproliferative neoplasms (MDS/MPN), respectively.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN75</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic II, Thomas R. and Athamanolap, Pornpat and Poh, Weijie and Chen, Chen and Hulbert, Alicia and Brock, Malcolm V. and Herman, James G. and Wang, Tza-Huei}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{DREAMing: a simple and ultrasensitive method for assessing intratumor epigenetic heterogeneity directly from liquid biopsies}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nucleic Acids Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{43}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e154-e154}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0305-1048}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/nar/gkv795}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1093/nar/gkv795}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="year">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Analyst</abbr></div> <div id="RN5" class="col-sm-9"> <div class="title">Quantum dots in diagnostics and detection: principles and paradigms</div> <div class="author"> Pisanic, T. R., Zhang, Y., and Wang, T. H. </div> <div class="periodical"> <em>Analyst</em>, 2014 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="&lt;Go%20to%20ISI&gt;://WOS:000336737900003" class="btn btn-sm z-depth-0" role="button">HTML</a> </div> <div class="abstract hidden"> <p>Quantum dots are semiconductor nanocrystals that exhibit exceptional optical and electrical behaviors not found in their bulk counterparts. Following seminal work in the development of water-soluble quantum dots in the late 1990’ s, researchers have sought to develop interesting and novel ways of exploiting the extraordinary properties of quantum dots for biomedical applications. Since that time, over 10 000 articles have been published related to the use of quantum dots in biomedicine, many of which regard their use in detection and diagnostic bioassays. This review presents a didactic overview of fundamental physical phenomena associated with quantum dots and paradigm examples of how these phenomena can and have been readily exploited for manifold uses in nanobiotechnology with a specific focus on their implementation in in vitro diagnostic assays and biodetection.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN5</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pisanic, T. R. and Zhang, Y. and Wang, T. H.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Quantum dots in diagnostics and detection: principles and paradigms}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Analyst}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{139}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2968-2981}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0003-2654}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1039/c4an00294f}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{&lt;Go to ISI&gt;://WOS:000336737900003}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Expert Rev. Mol. Diagn.</abbr></div> <div id="RN4" class="col-sm-9"> <div class="title">The promise of methylation on beads for cancer detection and treatment</div> <div class="author">Guzzetta, A. A.,  Pisanic, T. R., Sharma, P., Yi, J. M., Stark, A., Wang, T. H., and Ahuja, N. </div> <div class="periodical"> <em>Expert Review of Molecular Diagnostics</em>, 2014 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="&lt;Go%20to%20ISI&gt;://WOS:000340829200007" class="btn btn-sm z-depth-0" role="button">HTML</a> </div> <div class="abstract hidden"> <p>Despite numerous technical hurdles, the realization of true personalized medicine is becoming a progressive reality for the future of patient care. With the development of new techniques and tools to measure the genetic signature of tumors, biomarkers are increasingly being used to detect occult tumors, determine the choice of treatment and predict outcomes. Methylation of CpG islands at the promoter region of genes is a particularly exciting biomarker as it is cancer-specific. Older methods to detect methylation were cumbersome, operator-dependent and required large amounts of DNA. However, a newer technique called methylation on beads has resulted in a more uniform, streamlined and efficient assay. Furthermore, methylation on beads permits the extraction and processing of miniscule amounts of methylated tumor DNA in the peripheral blood. Such a technique may aid in the clinical detection and treatment of cancers in the future.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN4</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Guzzetta, A. A. and Pisanic, T. R. and Sharma, P. and Yi, J. M. and Stark, A. and Wang, T. H. and Ahuja, N.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The promise of methylation on beads for cancer detection and treatment}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Expert Review of Molecular Diagnostics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{845-852}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1473-7159}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1586/14737159.2014.943665}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{&lt;Go to ISI&gt;://WOS:000340829200007}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nanomedicine</abbr></div> <div id="RN7" class="col-sm-9"> <div class="title">In vivo nanoneurotoxicity screening using oxidative stress and neuroinflammation paradigms</div> <div class="author">Kim, Y., Kong, S. D., Chen, L. H.,  Pisanic, T. R., Jin, S., and Shubayev, V. I. </div> <div class="periodical"> <em>Nanomedicine-Nanotechnology Biology and Medicine</em>, 2013 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="&lt;Go%20to%20ISI&gt;://WOS:000324520900023" class="btn btn-sm z-depth-0" role="button">HTML</a> </div> <div class="abstract hidden"> <p>Iron oxide nanoparticles (IONPs) are promising neuroimaging agents and molecular cargo across neurovascular barriers. Development of intrinsically safe IONP chemistries requires a robust in vivo nanoneurotoxicity screening model. Herein, we engineered four IONPs of different surface and core chemistries: DMSA-Fe2O3, DMSA-Fe3O4, PEG-Fe3O4 and PEG-Au-Fe3O4. Capitalizing on the ability of the peripheral nervous system to recruit potent immune cells from circulation, we characterized a spatiotemporally controlled platform for the study of in vivo nanobiointerfaces with hematogenous immune cells, neuroglial and neurovascular units after intraneural IONP delivery into rat sciatic nerve. SQUID magnetometry and histological iron stain were used for IONP tracking. Among the IONPs, DMSA-Fe2O3 NPs were potent pro-apoptotic agents in nerve, with differential ability to regulate oxidative stress, inflammation and apoptotic signaling in neuroglia, macrophages, lymphocytes and endothelial cells. This platform aims to facilitate the development of predictive paradigms of nanoneurotoxicity based on mechanistic investigation of relevant in vivo bio-nanointerfaces. Published by Elsevier Inc.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN7</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kim, Y. and Kong, S. D. and Chen, L. H. and Pisanic, T. R. and Jin, S. and Shubayev, V. I.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{In vivo nanoneurotoxicity screening using oxidative stress and neuroinflammation paradigms}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nanomedicine-Nanotechnology Biology and Medicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1057-1066}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1549-9634}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.nano.2013.05.002}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{&lt;Go to ISI&gt;://WOS:000324520900023}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin. Chim. Acta</abbr></div> <div id="RN6" class="col-sm-9"> <div class="title">Extraction and processing of circulating DNA from large sample volumes using methylation on beads for the detection of rare epigenetic events</div> <div class="author">Keeley, B., Stark, A.,  Pisanic, T. R., Kwak, R., Zhang, Y., Wrangle, J., Baylin, S., Herman, J., Ahuja, N., Brock, M. V., and Wang, T. H. </div> <div class="periodical"> <em>Clinica Chimica Acta</em>, 2013 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="&lt;Go%20to%20ISI&gt;://WOS:000328306100031" class="btn btn-sm z-depth-0" role="button">HTML</a> </div> <div class="abstract hidden"> <p>The use of methylated tumor-specific circulating DNA has shown great promise as a potential cancer biomarker. Nonetheless, the relative scarcity of tumor-specific circulating DNA presents a challenge for traditional DNA extraction and processing techniques. Here we demonstrate a single tube extraction and processing technique dubbed "methylation on beads" that allows for DNA extraction and bisulfite conversion for up to 2 ml of plasma or serum. In comparison to traditional techniques including phenol chloroform and alcohol extraction, methylation on beads yields a 15- to 5-fold improvement in extraction efficiency. The technique results in far less carryover of PCR inhibitors yielding analytical sensitivity improvements of over 25-fold. The combination of improved recovery and sensitivity make possible the detection of rare epigenetic events and the development of high sensitivity epigenetic diagnostic assays. (C) 2013 Elsevier B.V. All rights reserved.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">RN6</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Keeley, B. and Stark, A. and Pisanic, T. R. and Kwak, R. and Zhang, Y. and Wrangle, J. and Baylin, S. and Herman, J. and Ahuja, N. and Brock, M. V. and Wang, T. H.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Extraction and processing of circulating DNA from large sample volumes using methylation on beads for the detection of rare epigenetic events}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinica Chimica Acta}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{425}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{169-175}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0009-8981}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cca.2013.07.023}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{&lt;Go to ISI&gt;://WOS:000328306100031}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Thomas Pisanic. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Last updated: June 26, 2023. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>